openedx/core/djangoapps/xblock/runtime/learning_core_runtime.py

```diff
@@ -291,6 +329,44 @@ def _lookup_asset_url(self, block: XBlock, asset_path: str) -> str | None:  # py
 
         This is called by the XBlockRuntime superclass in the .runtime module.
 
-        TODO: Implement as part of larger static asset effort.
+        TODO: Like get_block, we currently assume that we're using the Draft
+        version. This should be a runtime parameter.
```

Comment:
You probably know this, but [here in `apps.py`](https://github.com/openedx/edx-platform/blob/de7fa096673d2b38df1ae48e4b1e8e3de294f0b9/openedx/core/djangoapps/xblock/apps.py#L46-L75) is where we can set runtime parameters like "studio should default to DRAFT" and "LMS should default to PUBLISHED". 

It would be very cool if we could later override this default using a context manager when needed, but I don't know if that's a safe way to implement such an override.

Reply:
Is it sufficient to do Draft vs. Published, or should we be able to do any stored version (for the sake of seeing what changed)?

Reply:
Being able to see any stored version would be great, so I think we should do that. But more often I'll just want to specify "latest draft" or "latest published" so I hope we can have both options.